### PR TITLE
Task 4

### PR DESCRIPTION
**Pull request by Skysplit**
Add admin panel page

### CODE REVIEW

File: app/views/layouts/admin.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Admin Panel | Recruitment
+    </title>
+    <%= csrf_meta_tags %>
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (socjopata) [inline app/views/layouts/admin.html.erb]: :scissors: the whitespace

File: app/controllers/admin/application_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class Admin::ApplicationController < ApplicationController
+  layout 'admin'
+  before_action :authenticate_user!, :check_admin_access
+
+  private
```
Comment (socjopata) [inline app/controllers/admin/application_controller.rb]: Please add an empty line following the `private` definition :)

**Comment by Skysplit** [comment]
@socjopata done. Can I merge? :)

**Comment by socjopata** [comment]
Yep, I approved this PR 20 minutes ago ;)

**Pull request merged**